src/base/bittorrent/session.h

```diff
@@ -135,6 +135,14 @@ namespace BitTorrent
         };
         Q_ENUM_NS(SeedChokingAlgorithm)
 
+        enum class TLSVersion : int
```

Comment:
Also "TLS" appears two times in the resulting names (e.g. "TLSVersion::TLS11") that is redundant, IMO. Although it seems to be hard to avoid it in this case.

Reply:
But you can declare it as non scoped enum.

Reply:
Done.